tests/qa_automation/qa_run.pm

```diff
@@ -209,6 +214,9 @@ sub run() {
     my $junit_type = $self->junit_type();
     assert_script_run("/usr/share/qa/qaset/bin/junit_xml_gen.py -n '$junit_type' -d -o /tmp/junit.xml /var/log/qaset");
     parse_junit_log("/tmp/junit.xml");
+
+    # Switch back to root-console
+    select_console('root-console');
```

Comment:
It does not harm to re-select the current console. We also do this all the time, e.g. when starting a test module which is executing a `select_console` call in the beginning to be independant of previous modules. I would even consider this good style. If you see a problem with that approach we should rather within the backend make sure it's a noop.